documentation/modules/security/proc-renewing-ca-certs-manually.adoc

```diff
@@ -4,17 +4,17 @@
 
 [id='proc-renewing-ca-certs-manually-{context}']
 
-= Renewing CA certificates manually
+= Manually renewing the CA certificates generated by the Cluster Operator
 
-Cluster and clients CA certificates auto-renew at the start of their respective certificate renewal periods.
-If `Kafka.spec.clusterCa.generateCertificateAuthority` and `Kafka.spec.clientsCa.generateCertificateAuthority` are set to `false`,
-the CA certificates do not auto-renew.
-
-You can manually renew one or both of these certificates before the certificate renewal period starts.
+Cluster and clients CA certificates generated by the Cluster Operator auto-renew at the start of their respective certificate renewal periods.
+However, you can use the `strimzi.io/force-renew` annotation to manually renew one or both of these certificates before the certificate renewal period starts.
 You might do this for security reasons, or if you have xref:con-certificate-renewal-str[changed the renewal or validity periods for the certificates].
 
 A renewed certificate uses the same private key as the old certificate.
 
+NOTE: If you are using your own CA certificates, the renew annotation cannot be used.
```

Comment:
```suggestion
NOTE: If you are using your own CA certificates, the `force-renew` annotation cannot be used.
```

Reply:
Changed the related note for `force-replace` in the same way